bankayma_account/data/mail_template.xml

```diff
@@ -20,14 +20,12 @@
             </div>
         </field>
     </record>
-
     <record id="template_vendor_bill_validated" model="mail.template" forcecreate="1">
         <field name="name">Vendor bill rejected</field>
         <field name="model_id" ref="account.model_account_move" />
         <field name="subject">
-            [BanKayma]
-            <t t-out="object.company_id.name" />
-            - payment postponed for insufficient documents
+            [BanKayma] {{object.company_id.name}} - payment postponed for insufficient
+            documents
```

Comment:
@toyrobot note that qweb only is valid within views, for mail template fields you need jinja2